src/test/java/com/microsoft/azure/spring/data/cosmosdb/repository/MemoRepository.java

```diff
@@ -14,4 +15,6 @@
 @Repository
 public interface MemoRepository extends DocumentDbRepository<Memo, String> {
     List<Memo> findMemoByDate(Date date);
+
+    List<Memo> findMemoByMemoType(MemoType memoType);
```

Comment:
I think `findMemoByType` is good enough

Reply:
here should be align with field name in class, like `By${FieldName}`.

Reply:
As the field name in the domain is `MemoType`, so the method name must be `findMemoByMemoType`, as Type is a general class name, so I chose `MemoType` for more accurate semantic.